tests/integration/test_storage_s3_queue/test.py

```diff
@@ -2333,7 +2333,6 @@ def check_string_settings(node, settings):
         check_string_settings(node, string_settings)
 
         node.restart_clickhouse()
-        assert expected_rows == get_count()
```

Comment:
why did count become wrong after changing settings?

Reply:
There is `files_to_generate = 1000`, but we do alter `"tracked_files_limit": 50`, so we will process them again